waffle-cli/package.json

```diff
@@ -44,6 +44,7 @@
   "dependencies": {
     "@ethereum-waffle/chai": "^2.4.1",
     "@ethereum-waffle/compiler": "^2.4.1",
+    "@ethereum-waffle/doppelganger": "^0.0.1",
```

Comment:
Let's keep versioning across packages consistent and make it 2.4.1 (easier to release)

Reply:
In fact let's change all to `2.5.0`